srcCpp/Infrastructure_micro.cxx

```diff
@@ -51,7 +66,22 @@ unsigned long long PerftestClock::getTimeUsec()
             &clockSec,
             &clockUsec,
             (struct OSAPI_NtpTime*)&clockTimeAux);
-    return clockUsec + 1000000 * clockSec;
+    return clockUsec + (unsigned long long)1000000 * clockSec;
+
+#else
+    /*
+        The currect clock used by Micro on Windows is measured in milliseconds,
```

Comment:
comments:
```
/*
 * Line 1
 * Line 2
 */
```

Reply:
- Ask Francis is we should create a jira issue for this lack of precision on Windows, and if he agrees, add it here.
- Reference the Microsoft page where this way to measure time is documented.